From d0440b33c132692654dab2fc74cc8024e39f95a5 Mon Sep 17 00:00:00 2001
From: Chris Liddell <chris.liddell@artifex.com>
Date: Thu, 1 Apr 2021 08:19:05 +0100
Subject: [PATCH] Fix MacOS/tesseract build problems on Unix systems

Improve the configure test for valid C++ compiler options by having the
test include a header.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index a566fdbfe..d938d2dce 100644
--- a/configure.ac
+++ b/configure.ac
@@ -864,7 +864,7 @@ if test x$with_tesseract != xno; then
       for flag in $cxxflags_to_try ; do
         CXXFLAGS="$CXXFLAGS $flag"
 
-        AC_TRY_COMPILE(, [return 0;], [
+        AC_TRY_COMPILE([#include <cstdlib>], [return 0;], [
                 echo "   $flag"; CXXFLAGS_TO_USE="$CXXFLAGS_TO_USE $flag"
         ])
 
-- 
2.17.1
